### PR TITLE
Issue #623: Created new install, quickstart, and docker daemon topics

### PR DESCRIPTION
**Pull request by thatdocslady**
Fixes issue #623 

This PR includes 3 new topic files for install, quickstart, and docker daemon, plus updated topic metadata file with TOC definitions for these files. The information is originally from the readme.

Out of scope: README version of this content will not be removed for now, until we get the docs published. Then will handle as a part of issue #533.

### CODE REVIEW

File: docs/source/getting-started/installing.adoc [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+toc::[]
+
+The following section describes how to install Minishift and the required dependencies.
+
+[[install-prerequisites]]
+== Prerequisites
+
+Minishift requires a hypervisor to start the virtual machine containing
+OpenShift. Make sure that the hypervisor of your choice is installed and
```
Comment (LalatenduMohanty) [inline docs/source/getting-started/installing.adoc]: The virtual machines does not contain OpenShift. But it provisions OpenShift in the VM using "oc cluster up" . So I think the text should be `Minishift requires a hypervisor to start a virtual machine and it is used to provision single node OpenShift cluster.`

**Comment by hferentschik** [comment]
Restest this please.

**Comment by hferentschik** [comment]
I'll merge this after pull request #627 is merged.

**Comment by hferentschik** [comment]
Merged, thanks

**Comment by LalatenduMohanty** [comment]
@hferentschik Not sure if you have seen the review comment given by me. I was expecting my comment will be addressed and then we would merge the PR. Comments?

**Comment by thatdocslady** [comment]
@LalatenduMohanty I didn't get a chance to implement your comments before the PR was merged, but I can create a separate follow-up issue and fix it then.

**Comment by LalatenduMohanty** [comment]
@thatdocslady lets do that. Thanks

**Comment by hferentschik** [comment]
+1, my mistake. I missed this one. And the wording is indeed unfortunate. To be fair, this was exactly what we had before.

**Pull request closed without merging**